src/domain/usecases/utils/incident-action/GetResponseActionConfigurableForm.ts

```diff
@@ -29,3 +32,38 @@ export function getResponseActionConfigurableForm(
 
     return Future.success(incidentResponseActionData);
 }
+
+export function getSingleResponseActionConfigurableForm(options: {
+    eventTrackerDetails: DiseaseOutbreakEvent;
+    responseActionId: string;
+    configurations: Configurations;
+}): FutureData<SingleResponseActionFormData> {
+    const { eventTrackerDetails, responseActionId, configurations } = options;
+    const { incidentResponseActionConfigurations } = configurations.selectableOptions;
+
+    const incidentResponseActionData: SingleResponseActionFormData = {
+        type: "incident-response-action",
+        eventTrackerDetails: eventTrackerDetails,
+        entity:
+            eventTrackerDetails.incidentActionPlan?.responseActions.find(
+                responseAction => responseAction.id === responseActionId
```

Comment:
do not throw an error if the response action if not found. return "undefined" and in the presentation layer, if the formtype is SingleResponseActionConfigurableForm and entity is undefined, then show error in snackbar.